app/views/unattended/provisioning_templates/snippet/remote_execution_ssh_keys.erb

```diff
@@ -59,8 +59,8 @@ EOF
 
 <% if ssh_user != 'root' && host_param('remote_execution_effective_user_method') == 'sudo' -%>
 <% if @host.operatingsystem.family == 'Redhat' || @host.operatingsystem.family == 'Debian' -%>
-echo "<%= ssh_user %> ALL = (root) NOPASSWD : ALL
-Defaults:<%= ssh_user %> !requiretty" > /etc/sudoers.d/<%= ssh_user %>
+echo "<%= ssh_user %> ALL = (root) NOPASSWD : ALL" > /etc/sudoers.d/<%= ssh_user %>
+echo "Defaults:<%= ssh_user %> !requiretty" >> /etc/sudoers.d/<%= ssh_user %>
 <% elsif @host.operatingsystem.family == 'Suse' -%>
 echo "<%= ssh_user %> ALL = (root) NOPASSWD : ALL
```

Comment:
@maximiliankolb I still think we should apply the same fix here because it's pretty much guaranteed to also be an issue.

Reply:
OK, applied for SLES too.